docs/dataservice-write-example.md

```diff
@@ -77,6 +77,28 @@ After installing and running the `dataservice_write_example` apk, you should see
 
 Note, that you can run `CMake` command directly from `<olp-edge-sdk-root>/examples/dataservice-write/` folder if you have already built and installed OLP C++ EDGE SDK libraries for Android. Make sure that you pass the correct path to `LevelDB` library and provided the correct `EDGE_SDK_NETWORK_PROTOCOL_JAR` parameter in the `CMake` command invoked by the `build/examples/dataservice-write/android/app/build.gradle` script.
 
+## Building and running on iOS
+
+This example shows how to integrate and use the OLP C++ EDGE SDK in a basic iOS application written in Objective-C language.
+
```

Comment:
Please follow same structure as for Android.
Means:
### Prerequisities
### Build...
### Assemble... & Additional notes [IF NEEDED]

Reply:
Done